src/test/java/net/consensys/orion/enclave/sodium/SodiumEnclaveTest.java

```diff
@@ -164,6 +172,34 @@ void encryptGeneratesDifferentCipherForSamePayloadAndKey() {
     assertNotEquals(encryptedPayload1.cipherText(), encryptedPayload2.cipherText());
   }
 
+  @Test
+  public void generatesLegacyPrivacyGroupIdForDuplicateValues() {
+    final String expectedPrivacyGroupId = "/xzRjCLioUBkm5LYuzll61GXyrD5x7bvXzQk/ovJA/4=";
```

Comment:
nit: add a note for what this value represents? Or is it just magical?

Reply:
added a note